eth_utils/address.py

```diff
@@ -119,7 +119,7 @@ def is_same_address(left: AnyAddress, right: AnyAddress) -> bool:
         return to_normalized_address(left) == to_normalized_address(right)
 
 
-def to_checksum_address(value: AnyStr) -> Address:
+def to_checksum_address(value: AnyStr) -> HexAddress:
```

Comment:
Hrm, this actually looks like it should be `ChecksumAddress` as the return type.

Reply:
Ahh, yup!! Made the change